pkg/controller/backupschedule/backup_schedule_controller.go

```diff
@@ -139,10 +139,13 @@ func (bsc *Controller) processNextWorkItem() bool {
 	if err := bsc.sync(key.(string)); err != nil {
 		if perrors.Find(err, controller.IsRequeueError) != nil {
 			glog.Infof("BackupSchedule: %v, still need sync: %v, requeuing", key.(string), err)
+			bsc.queue.AddRateLimited(key)
+		} else if perrors.Find(err, controller.IsIgnoreError) != nil {
+			glog.V(4).Infof("BackupSchedule: %v, ignore err: %v, waiting for the next sync", key.(string), err)
```

Comment:
```suggestion
			glog.Infof("BackupSchedule: %v, ignore err: %v, waiting for the next sync", key.(string), err)
```
?

Reply:
When the backupSchedule is paused, every sync will print this log, which is a known behavior, so if you do not enable higher-level log output, it will not output this log.